src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -1266,7 +1266,8 @@ public Window GetWindow()
                 return window;
             }
 
-            window = (Registry.GetManagedBaseHandleFromNativePtr(Interop.Application.GetWindow(SwigCPtr)) as Window) ?? new Window(Interop.Application.GetWindow(SwigCPtr), true);
+            var nativeWindow = Interop.Application.GetWindow(SwigCPtr);
+            window = (Registry.GetManagedBaseHandleFromNativePtr(nativeWindow) as Window) ?? new Window(nativeWindow, true);
```

Comment:
Don't we need to DeleteWindow when we success to get `(Registry.GetManagedBaseHandleFromNativePtr(nativeWindow) as Window)` ?

The result of `Registry` and `nativeWindow` is not matched. so.. we may need to reduce 1 refcount whom `GetWindow` make increate.

Reply:
Correct. True. Good point. Done.
Thank you.